common/templates/course_modes/choose.html

```diff
@@ -71,19 +71,44 @@ <h3 class="title">${_("Sorry, there was an error when trying to enroll you")}</h
                             <div class="register-choice register-choice-certificate">
                                 <div class="wrapper-copy">
                                     <span class="deco-ribbon"></span>
+                                    % if has_credit_upsell:
+                                    <h4 class="title">${_("Pursue Academic Credit with a Verified Certificate")}</h4>
+
+                                    <div class="copy">
+                                        <p>${_("Become eligible for academic credit and highlight your new skills and knowledge with a verified certificate. Use this valuable credential to qualify for academic credit from {org}, advance your career, or strengthen your school applications.").format(org=course_org)}</p>
+                                        <p>
+                                            <div class="wrapper-copy-inline">
+                                                <div class="copy-inline">
+                                                    <h4>${_("Benefits of a Verified Certificate")}</h4>
+                                                    <ul>
+                                                        <li><b>${_("Eligible for credit")}: </b>${_("Receive academic credit after successfully completing the course")}</li>
+                                                        <li><b>${_("Official")}: </b>${_("Receive an instructor-signed certificate with the institution's logo")}</li>
+                                                        <li><b>${_("Easily shareable")}: </b>${_("Add the certificate to your CV, resume, or post it directly on LinkedIn")}</li>
```

Comment:
This line should be "Easily shareable: Add the certificate to your CV or resume, or post it directly on LinkedIn"